diff.py

```diff
@@ -973,6 +973,13 @@ def serialize(text: Optional[Text]) -> List[Dict[str, Any]]:
                     output_row[column_name] = column
             output_rows.append(output_row)
         output["rows"] = output_rows
+        return output
```

Comment:
The return type annotation of `table()` becomes wrong now. Can you rename the method to something else, leaving a `table` method that just raises a NotImplementedError?

(mypy ought to catch this error, I believe)